frontend/src/main/components/Shift/ShiftTable.js

```diff
@@ -0,0 +1,71 @@
+
+import OurTable, { ButtonColumn } from "main/components/OurTable"
+import { useBackendMutation } from "main/utils/useBackend";
+
+
+
+export default function ShiftTable({ shift }) {
+
+    function cellToAxiosParamsToggleAdmin(cell) {
+        return {
+            url: "/api/admin/users/toggleAdmin",
+            method: "POST",
+            params: {
+                id: cell.row.values.id
+            }
+        }
+    }
+
+    // Stryker disable all : hard to test for query caching
+    const toggleAdminMutation = useBackendMutation(
+        cellToAxiosParamsToggleAdmin,
+        {},
+        ["/api/admin/users"]
+    );
+    // Stryker enable all 
```

Comment:
Please change to `// Stryker restore all`

See: https://ucsb-cs156-s23.slack.com/archives/C050779M19P/p1685726928710669